data.sql

```diff
@@ -1,6 +1,47 @@
-/* Populate database with sample data. */
+INSERT INTO
+  animals(
+  	id,
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES
+  (1, 'Agumon', '2022-02-03', 0, true, 10.23);
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
-.
\ No newline at end of file
+INSERT INTO
+  animals(
+  	id,
```

Comment:
- [OPTIONAL] Instead of hardcoding our id values,  it will be nice to make the id column not null and auto-incremented like so: `id int NOT NULL IDENTITY(1,1)`.

Reply:
@Mihndim2020  Thank you for your review and suggestion.